android/src/main/java/com/stripeterminalreactnative/Mappers.kt

```diff
@@ -446,7 +446,7 @@ internal fun mapFromReaderSoftwareUpdate(update: ReaderSoftwareUpdate?): Writabl
                 "estimatedUpdateTime",
                 mapFromUpdateTimeEstimate(it.timeEstimate)
             )
-            putString("requiredAt", convertToUnixTimestamp(it.requiredAt.time))
```

Comment:
is there a reason we are fixing here rather than remove the `* 1000` part in https://github.com/stripe/stripe-terminal-react-native/blob/93f3ca0b/android/src/main/java/com/stripeterminalreactnative/Mappers.kt#L439

Reply:
After function conversion, the "created" field on PaymentIntents is converted to same time units milliseconds on both Android and iOS except requiredAt. So we only need to fixing here.